CMakeLists.txt

```diff
@@ -55,6 +54,24 @@ ADD_DEFINITIONS(-DPROJECT_SOURCE_DIR=\\"${CMAKE_SOURCE_DIR}\\"
 # add the 'lib' dir to cmake's module search path
 SET(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/lib/")
```

Comment:
Seems not needed anymore since line 58 is added:

    list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/lib/")

Reply:
Ok